tests/src/JIT/Directed/intrinsic/pow/pow1.cs

```diff
@@ -16,19 +16,18 @@ public static int Main()
         //Check if the test is being executed on ARM
         bool isProcessorArm = false;
 
+        string processorArchEnvVar = null;
+
 #if CORECLR 
-        if (TestLibrary.Env.GetEnvVariable("PROCESSOR_ARCHITECTURE").Equals(
-                   "ARM", StringComparison.CurrentCultureIgnoreCase))
-        {
-            isProcessorArm = true;
-        }
+        processorArchEnvVar = TestLibrary.Env.GetEnvVariable("PROCESSOR_ARCHITECTURE");
```

Comment:
BTW: GetEnvironmentVariable is available in .NET Core profile. This ifdef should not be necessary.